runelite-client/src/main/java/net/runelite/client/plugins/customcursor/CustomCursorConfig.java

```diff
@@ -40,4 +41,15 @@ default CustomCursor selectedCursor()
 	{
 		return CustomCursor.RS3_GOLD;
 	}
+
+	@ConfigItem(
+		position = 1,
+		keyName = "bigScreenStyle",
+		name = "Big screen mode",
```

Comment:
I think this feature would be much better received if it were just a simple "Large cursor" toggle, which didn't depend on screen size. I can imagine some players wanting the cursor to be double-sized even at smaller resolutions as a matter of personal preference. In addition, I don't really see a benefit to restricting it to larger screens in this way.